helm-chart/binderhub/values.yaml

```diff
@@ -238,8 +238,8 @@ dind:
 imageCleaner:
   enabled: true
   image:
-    name: jupyterhub/k8s-image-cleaner
-    tag: 'local'
+    name: quay.io/jupyterhub/docker-image-cleaner
```

Comment:
Can we automate updating these with dependabot?

Reply:
Not with dependabot, but i can make it happen without. I figure i want that done for chp and traefil in z2jh as well.

Reply:
To do this, I imagine a scan for the latest git tag and an automatic PR creation to bump to the latest git tag.

Reply:
@manics depends, what if they push many tags, also for backports? Also, what kind of API's are easy to use to get such tags that makes us able to re-use them and not have custom logic for each image version updater because we work against both dockerhub and quay etc?

I opened https://github.com/jupyterhub/zero-to-jupyterhub-k8s/issues/2419 in z2jh about bumping things. I'll open another one to mimic what we do there in the end for this repo.

Reply:
I created https://github.com/jupyterhub/binderhub/issues/1398